### PR TITLE
[ORCA-488] Add Conftest Run implementation

### PR DESCRIPTION
**Pull request by nishkrishnan**
With this PR, we have the ability to run local policies. The output doesn't get posted to the PR, but the commit status will pass/fail accordingly 🎉 

Had to change a couple things around for ensuring the version.  
* Needed to use go-getter `GetAny` since conftest downloads as a directory
* Because of the above I couldn't rename it on the download itself and had to change the loaderpath to a nonconflicting one for each version.
* Then I had to symlink that with the correct serialized name

Also I created `Dockerfile.testenv` which i pushed to my personal dockerhub account to quickly unblock testing which was failing since conftest isn't in the image, and we prevent downloading it.  We should get this into a lyft organization or in the main testing image.

### CODE REVIEW

File: server/events/command_runner.go [hunk 1]
```diff
@@ -533,6 +533,10 @@ func (c *DefaultCommandRunner) runProjectCmds(cmds []models.ProjectCommandContex
 		case models.ApplyCommand:
 			res = c.ProjectCommandRunner.Apply(pCmd)
 		}
+
+		if res.Error != nil {
+			pCmd.Log.Err("Error running project command %s", res.Error.Error())
```
Comment (msarvar) [inline server/events/command_runner.go:538]: Don't you want to add this to parallel runner as well?

File: server/events/runtime/policy/conftest_client.go [hunk 2]
```diff
@@ -104,12 +148,40 @@ func NewConfTestExecutorWorkflow(log *logging.SimpleLogger, versionRootDir strin
...
 		},
+		Exec: runtime_models.LocalExec{},
 	}
 }
 
-func (c *ConfTestExecutorWorkflow) Run(log *logging.SimpleLogger, executablePath string, envs map[string]string, args []string) (string, error) {
-	return "success", nil
+func (c *ConfTestExecutorWorkflow) Run(ctx models.ProjectCommandContext, executablePath string, envs map[string]string, workdir string) (string, error) {
+	policyArgs := []Arg{}
+	for _, policySet := range ctx.PolicySets.PolicySets {
```
Comment (msarvar) [inline server/events/runtime/policy/conftest_client.go:157]: nice PolicySets :D

**Review by msarvar — APPROVED** [review]
This is great!

File: server/events/command_runner.go (hunk 1 above)
Comment (nishkrishnan) [reply server/events/command_runner.go:538]: hmm i actually might remove this for here and submit as an upstream fix with the parallel runner one as well.

**Review by msarvar — APPROVED** [review]

**Pull request merged**